far/viewer.cpp

```diff
@@ -296,13 +296,29 @@ wchar_t Viewer::ZeroChar() const
 
 void Viewer::ChangeHexModeBytesPerLine(int const Amount)
 {
-	const auto NewValue =
-		std::clamp(m_HexModeBytesPerLine + Amount, s_HexModeBytesPerStripe, hex_mode_bytes_from_line_width(ScrollbarAdjustedWidth, get_char_size(m_Codepage)));
+	if (Amount == 0) return;
```

Comment:
I think this function is supposed to be called only with ±(1, 16), maybe just assert?

Reply:
It works for any (reasonable) `Amount` without sacrificing readability (I hope) and performance (almost). We may want to add snapping 8-byte grid or reuse the code elsewhere. Let's keep it generic.